Update Toast component docs (#3513)
skeletonlabs/skeleton: sites/skeleton.dev/src/content/docs/components/toast/svelte.mdx

import ExamplePromise from '@examples/components/toast/ExamplePromise.svelte';
import ExamplePromiseRaw from '@examples/components/toast/ExamplePromise.svelte?raw';

The first example showcases the complete setup process for a reusable toast component, while the following are abridged to handle everything in a single file. Please keep this in mind when following these examples.

<Preview client:load>
	<Fragment slot="preview">
		<Example client:visible />
	</Fragment>
	<Fragment slot="code">
		<div class="space-y-4">
			Import `createToaster()` and export a new shared reference.

> 			<Code code={ToasterReuseUtil} lang="ts" title="toaster-svelte.ts" />

Reviewer: ```suggestion
			<Code code={ToasterReuseUtil} lang="ts" title="toaster.ts" />
```

Author: (ditto to above)

> 			Import the `<Toaster>` component and `appToaster` reference in your root layout.

Reviewer: ```suggestion
			Import the `<Toaster>` component and `toaster` reference in your root layout.
```

			<Code code={ExampleReuseLayoutRaw} lang="svelte" title="+layout.svelte" />

> 			Import `appToaster` reference and use it to trigger toasts anywhere in your client-side app.

Reviewer: ```suggestion
			Import `toaster` reference and use it to trigger toasts anywhere in your app.
```
